### DISCUSSION TITLE
Terminated (ADAPTER_REQUESTED_TERMINATION): Login to KLF-200 device at http://VELUX-KLF-EAF0 failed.

### DISCUSSION THREAD

**Discussion by mitenheoh** [MiSchroe/ioBroker.klf200#117] [Q&A] [answered]
Hallo MiSchroe,

nach heute erfolgreicher Installation des Adapters nun das Problem, dass der Login nicht funktioniert:

- Die KLF200 hängt am LAN direkt am Router ohne Switch o.ä.
- Im Router sehe ich den KLF aktiv mit IP-Adresse und Name VELUX-KLF-EAF0 (Das ähnelt der auf der Rückseite aufgedruckten SSID VELUX_KLF_AA95); auf ein PING reagiert der KLF200 einwandfrei
- Über die Website des KLF200 wurde die o.g. IP-Adresse fest eingestellt, d.h. diese wurde nicht vom DHCP festgelegt, was zeigt, dass KLF200 und Router kommunizieren
- Als Passwort ist das WLAN-Passwort eingetragen, welches auf der Rückseite als Password dargestellt ist (Genau genommen aus der Website des KLF200 kopiert, d.h. selbst Tipp- oder Lesefehler sind da ausgeschlossen)
- Als Host wurden verschiedenste Schreibweisen ausprobiert, aber alle führen zu "Login to KLF-200 device at ... failed"

1. http://VELUX-KLF-EAF0
2. http://velux-klf-eaf0
3. VELUX-KLF-EAF0
4. velux-klf-eaf0
5. https://VELUX-KLF-EAF0
6. https://velux-klf-eaf0
7. http://VELUX-KLF-AA95
8. http://velux-klf-aa95
9. VELUX-KLF-AA95
10. velux-klf-aa95
11. https://VELUX-KLF-AA95
12. https://velux-klf-aa95
13. http://klf200.velux
14. http://192.168.0.137

Da mir das Host-Feld das einzige zu sein scheint, bei dem Gestaltungsspielraum besteht, würde ich mich über Hilfe freuen, was ich dort eintragen muss.

Vielen Dank und schöne Grüße
Tim

PS: Ich bin echt gespannt, das alles in Aktion zu erleben! Ich bin froh, dass es den Adapter gibt! Danke.

**Comment by MiSchroe** [comment, marked as answer]
Hallo Tim,

in das Feld Host-Name bitte nur den Namen eintragen ohne `https://`, also in deinem Fall `velux-klf-eaf0`. Alternativ kannst du dort auch die IP-Adresse eintragen, also z.B. `192.168.0.137`. Evtl. musst du hinter dem Namen noch einen Netzwerknamen angeben, ich muss z.B. `velux-klf-xxxx.fritz.box` eintragen, damit die Namensauflösung funktioniert. Mit der IP-Adresse geht es aber immer, denn dann muss der Name nicht aufgelöst werden.

Als Passwort wird das WLAN-Passwort verwendet, aber wenn der Router vorher noch die alte Firmware installiert hatte, kann es auch das Passwort der Web-Oberfläche sein (so ist es z.B. bei mir). Am besten einfach mal mit beiden Passwörtern ausprobieren.

Viele Grüße
Michael

**Reply by mitenheoh** [reply]
Hallo Michael,

vielen Dank für die schnelle Rückmeldung!

Es gab noch einige Fehlerversuche, obwohl ich immer alles richtig hatte, denke ich.

Den Unterschied hat offenbar gemacht, dass ich im IO-Broker alles korrekt eingestellt hatte (als Host die IP) und dann den KLF200 nochmal vom LAN weggenommen und danach am LAN neu gestartet habe. Das war wohl der Durchbruch, dass sich beide Teile "finden" konnten.

Jetzt bin ich gespannt, den Adapter im Einsatz zu erleben!
Beste Grüße
Tim
